### ISSUE TITLE
Website down

### ISSUE THREAD

**Issue by winterscar** [system-ui/system-ui.com#9]
The system-ui.com website seems to be incorrectly configured or down:

<img width="502" alt="image" src="https://user-images.githubusercontent.com/11340175/175332528-f5fdf094-e9a8-4e78-bb85-182cecdfe7b3.png">

**Comment by lachlanjc** [comment]
Oh no! I'll reach out to Brent since no one else can fix that, but I've got the website deployed at https://system-ui.vercel.app/ in the meantime.

**Comment by C0D3-BL00D3D** [comment]
Followed a link from the [AWS Amplify UI docs](https://ui.docs.amplify.aws/react/theming#design-tokens) to the [system-ui.com](https://system-ui.com/) website, and it looks like it's still down with the same error as above.

**Comment by lachlanjc** [comment]
Yup! I've reached out to the domain owner but unfortunately there's nothing we can do to fix it in the meantime :(

**Comment by mix3d** [comment]
9 months later, still nothing? Also followed a link from a Salesforce Composable Storefront / PWAKit page.

**Comment by quangv** [comment]
https://system-ui.vercel.app/ - website is down as well 😕

**Comment by lachlanjc** [comment]
Fixed, sorry about that! Not sure why the subdomain dropped off the list.
